src/components/Comment.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { CommentInt } from '../types/CommentInt';
+import { deleteComment } from '../utils/RESTmethods';
+
+type CommentProps = {
+  comment: CommentInt,
+  setComments: Dispatch<SetStateAction<CommentInt[]>>,
+};
+
+export const Comment: React.FC<CommentProps> = ({
+  comment,
+  setComments,
+}) => {
+  const {
+    email,
+    name,
+    body,
+    id,
+  } = comment;
+
+  const handleDeleteComment = (commentId: number) => {
+    setComments(prevCom => [...prevCom].filter(com => com.id !== commentId));
```

Comment:
No need to create copy, filter return new array and do not mutate original
```suggestion
    setComments(prevCom => prevCom.filter(com => com.id !== commentId));
```